bundles/org.openhab.binding.dsmr/src/main/resources/OH-INF/thing/meter_electricity_emucs_v1_0.xml

```diff
@@ -21,6 +21,8 @@
 			<channel id="emeter_tariff_indicator" typeId="tariffIndicatorType"/>
 			<channel id="emeter_actual_delivery" typeId="actualDeliveryType"/>
 			<channel id="emeter_actual_production" typeId="actualProductionType"/>
+			<channel id="emeter_actual_demand" typeId="actualDeliveryBelgiumType"/>
+			<channel id="emeter_maximum_demand_current_month" typeId="actualDeliveryBelgiumType"/>
```

Comment:
Either make a separate channel type for current month or set different label on channel. That way the user can see the difference in the UI. Otherwise they will show with the same text in the UI.
```suggestion
			<channel id="emeter_maximum_demand_current_month" typeId="actualDeliveryBelgiumType">
			   <label>Power Delivery Current Month</label>
			 </channel>
```
Also add these new texts to the [dsmr.properties](https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.dsmr/src/main/resources/OH-INF/i18n/dsmr.properties) file. 
You can do this by calling `mvn i18n:generate-default-translations -pl :org.openhab.binding.dsmr`.